rmgpy/chemkin.pyx

```diff
@@ -73,10 +73,10 @@ def readThermoEntry(entry, Tmin=0, Tint=0, Tmax=0):
     the label of the species and the thermodynamics model as a :class:`NASA`
     object.
     
-    Format specification at http://www2.galcit.caltech.edu/EDL/public/formats/chemkin.html
+    For format specification, see the Chemkin Input Manual
     """
     lines = entry.splitlines()
-    species = str(lines[0][0:18].split()[0].strip())
+    species = str(lines[0][0:22].split()[0].strip())
```

Comment:
The thermo specification has columns 1-16 for the species name and columns 19-24 for a comment.

Was there a reason you chose 22? I think it could probably be safely changed to 24.

Reply:
Just to clarify a bit, if we're reading external chemkin files, we should still be able to parse the identifier and comment properly. If we're reading RMG chemkin files where we might have an extra long identifier, we might as well use the full 24 characters. If we're writing chemkin files, using 22 characters already breaks chemkin compatibility, so it won't matter if we use 24. So this can be changed at lines 1470 and 1968 as well.

Reply:
Good idea. I modified it to allow longer strings here.